outpost/pkg/ldap/instance_search.go

```diff
@@ -137,3 +159,32 @@ func (pi *ProviderInstance) GroupEntry(g api.Group) *ldap.Entry {
 	dn := pi.GetGroupDN(g)
 	return &ldap.Entry{DN: dn, Attributes: attrs}
 }
+
+func (pi *ProviderInstance) UserAsGroupEntry(u api.User) *ldap.Entry {
+	dn := fmt.Sprintf("cn=%s,%s", u.Username, pi.GroupDN)
+
+	attrs := []*ldap.EntryAttribute{
+		{
+			Name:   "cn",
+			Values: []string{u.Username},
+		},
+		{
+			Name:   "uid",
+			Values: []string{string(u.Uid)},
+		},
+		{
+			Name:   "objectClass",
+			Values: []string{GroupObjectClass, "goauthentik.io/ldap/group"},
```

Comment:
Maybe we can add another objectClass to indicate that this is a fake group, like `goauthentik.io/ldap/virtual-group`?

Reply:
Instead of `goauthentik.io/ldap/group` or in addition to?

Reply:
I'd say in addition.

Actually, on further inspection, wouldn't it be possible to use `pi.GroupEntry()` and pass and a custom Group object?

Reply:
Not a bad idea, I'll do that

Reply:
I've ran makemigrations, lint-fix and added the fields to the LDAPProviderForm now